src/main/kotlin/gg/octave/bot/commands/music/dj/Shuffle.kt

```diff
@@ -25,4 +28,14 @@ class Shuffle : MusicCog {
         manager.scheduler.shuffle()
         ctx.send("Player has been shuffled")
     }
+
+    @SubCommand(description = "Toggle whether the queue should be shuffled before every play.")
```

Comment:
Adjust description to new logic.
```suggestion
    @SubCommand(description = "Toggle whether the queue should be shuffled when songs are added.")
```